### ISSUE TITLE
Documentation about jobs that run during e2e test cases.

### ISSUE THREAD

**Issue by LochanRn** [kubernetes-sigs/cluster-api-provider-azure#1497]
/kind documentation

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]
Documentation about each job in e2e test cases. A short description about what each job does.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- cluster-api-provider-azure version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

**Comment by CecileRobertMichon** [comment]
Something similar to https://github.com/kubernetes-sigs/cluster-api/pull/4899

**Comment by sayantani11** [comment]
@CecileRobertMichon Is the issue still open?

**Comment by CecileRobertMichon** [comment]
@sayantani11 yes it is, would love some help with it if you want to work on it! Let us know if you need help getting started

**Comment by sayantani11** [comment]
@CecileRobertMichon Yeah I would like to work on it, & yes can you help me on how to start?

**Comment by LochanRn** [comment]
@sayantani11  Check these out for you to get started, may be something similar to this would be great :).
https://github.com/kubernetes-sigs/cluster-api/issues/4665
https://cluster-api.sigs.k8s.io/reference/jobs.html
